model/project_ref.go

```diff
@@ -2434,6 +2434,44 @@ func GetMessageForPatch(patchID string) (string, error) {
 	return project.CommitQueue.Message, nil
 }
 
+func ValidateContainers(project string, containers []Container) error {
+	pRef, err := FindMergedProjectRef(project, "", true)
+	if err != nil {
+		return errors.Errorf("Unable to find merged project ref for project %s", project)
+	}
+	catcher := grip.NewSimpleCatcher()
+	for _, container := range containers {
+		catcher.Add(ValidateContainerSystem(container.System))
+		catcher.Add(ValidateContainerSize(container.Resources))
+		catcher.Add(ValidateContainerSize(pRef.ContainerSizes[container.Size]))
```

Comment:
On an unrelated note, since this is checking the project ref for correctness of its defined container sizes, is it more appropriate to put this check as part of `projectSettingsValidators` instead of this (which is under `projectErrorValidators`)?

Reply:
Sure, that makes sense